slash/utils/traceback_proxy.py

```diff
@@ -0,0 +1,169 @@
+import sys
```

Comment:
Please add a docstring explaining why this hack is needed and what it does, even in brief